retrofit2-clients/src/test/java/com/palantir/remoting2/retrofit2/TestService.java

```diff
@@ -51,4 +51,7 @@
     // Note: Retrofit2 will discard any baseUrl path when the endpoint path starts with "/"
     @GET("/absolute")
     Call<String> getAbsolute();
+
+    @GET("/Api")
+    Call<String> getAbsoluteApiTitleCase();
```

Comment:
I need an absolute url with uppercase letters in it in order to test this correctly :/